### ISSUE TITLE
Inputs set in project settings do not take effect when playing in editor

### ISSUE THREAD

**Issue by vinceTheProgrammer** [Facepunch/sbox-issues#5906]
### Describe the bug

Input buttons defined in the editor project settings do not take effect when playing the game in the editor environment.

### To Reproduce

1. Create brand new project
2. Create GameObject
3. Add script to GameObject
4. Define a logger in the script
```cs
private Logger Logger = new Logger( "InputThing: " );
```
5. Add the following to an update method in the script:
```cs
Logger.Info( "Forward button origin: " + Input.GetButtonOrigin( "Forward" ) );
if (Input.Down("Forward"))
{
	GameObject.Transform.Position += new Vector3( 0, (float) 0.1, 0 );
}
```
6. Go to  `Project > Settings > Input > Movement` and change the forward input to a key other than "w", such as "k".
7. Play the game and observe that even though the button origin is correctly shown to be the new key you set, only the "w" key actually works to move the GameObject.

----
Minimal reproduction: https://github.com/vinceTheProgrammer/input-bug-reproduction

### Expected behavior

I expect that if I set a button for a particular input in the project's settings, it should reflect properly when testing the game in the editor.

### Media/Files

![Screenshot 2024-07-23 203918](https://github.com/user-attachments/assets/1be26f37-3f4e-4d5f-94d5-95e2e34eeca2)

### Additional context

- Running on Windows 10
- Using a Redragon K580RGB keyboard
- Someone else was able to reproduce the issue
- Initially found this behavior for the "Run" input, where "Run" always requires the SHIFT key.

**Comment by DevulTj** [comment]
I have a feeling this is because you're trying to overwrite the common input actions that have local preferences

**Comment by vinceTheProgrammer** [comment]
> I have a feeling this is because you're trying to overwrite the common input actions that have local preferences

Yeah, that's probably it because if I make an input action with a different name, for example "Run_c" instead of "Run", I can bind it to whatever just fine.

Is that the intended behavior though? It seems a little counterintuitive to ignore the bindings the dev sets without at least warning the dev that their bindings are being overwritten since the input action is part of the set of common input actions.

**Comment by DevulTj** [comment]
I don't believe this is happening anymore (?), can you verify
